server/src/cloud/board.ts

```diff
@@ -237,6 +237,42 @@ export const initializeBoardFunctions = () => {
     return true;
   });
 
+  api<{board: string}, boolean>("acceptAllPendingJoinRequests", async (user, request) => {
```

Comment:
I think it would be safer to extend the `acceptUser` endpoint by a variable list of users that should be accepted. In this implementation the admin might accidentally accept a user that wasn't shown in his list. It's an edge case, yet a possible vulnerability. 🤷‍♂️

Reply:
Excellent idea, changed it immediately 👍🏼